api-docs/common-gs/using-client.rst

```diff
@@ -78,11 +78,13 @@ Information about the environment variables is provided after the example.
 
 .. code::
 
-     export OS_AUTH_URL=https://identity.api.rackspacecloud.com/v2.0/
-     export OS_USERNAME=yourUserName
-     export OS_TENANT_ID=yourTenantId
-     export OS_REGION_NAME=yourRegionName
-     export OS_PASSWORD=yourPassword
+	 export OS_AUTH_URL=https://identity.api.rackspacecloud.com/v2.0/
+	 export OS_USERNAME=yourUserName
+	 export OS_TENANT_ID=yourTenantId
+	 export OS_REGION_NAME=yourRegionName
+	 export OS_PASSWORD=yourPassword
+
```

Comment:
Delete this blank line.  In the issue request, I think the blank line was intended to clarify what needed to be added.